src/gateways/createBookConfigLoader.ts

```diff
@@ -44,3 +44,5 @@ export default (baseUrl: string) => {
     },
   };
 };
+
+export const getBookVersionFromUUID = (uuid: string): BookVersion | undefined => cachedBooks[uuid];
```

Comment:
I would call this function differently and change the type of the first one to always return a promise